homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

```diff
@@ -0,0 +1,180 @@
+"""Support for Atlantic Pass APC DHW."""
+
+import logging
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.water_heater import (
+    STATE_ECO,
+    STATE_OFF,
+    STATE_PERFORMANCE,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from .. import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_OPERATION_MODE: dict[str, str] = {
+    STATE_PERFORMANCE: STATE_PERFORMANCE,
+    STATE_ECO: OverkizCommandParam.MANUAL_ECO_ACTIVE,
+    OverkizCommandParam.MANUAL: OverkizCommandParam.MANUAL_ECO_INACTIVE,
+}
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
You should import LOGGER from `const.py`.

Reply:
Sorry. Forgot to delete the logger after debugging. Done